### ISSUE TITLE
UI: toolbar buttons should not use "raised" style

### ISSUE THREAD

**Issue by bertob** [flxzt/rnote#349]
The tool chooser and some of the toolbar buttons are currently using the "raised" style. The HIG recommends not doing so, because within the context of a toolbar the extra border doesn't add anything (raised is mostly for free-floating and text-only buttons) and makes the interface less legible. Even back in the GNOME 2 days, toolbar buttons never had borders/backgrounds :)

Having a background only on active/pressed buttons (normal "flat" styling) would look cleaner and be more consistent with other apps.

![image](https://user-images.githubusercontent.com/1908896/201684589-57ffdbee-7a81-4d21-832c-6c5d535be1e2.png)

**Comment by flxzt** [comment]
Agree for the header tools chooser, adding `toolbar` to it makes it consistent to the narrow tools chooser.

In the sidbear I am just using the default adwaita style though, I would need to add the `flat` style to all buttons. This is how it looks with it added to all normal and menu buttons, the linked toggle buttons and the spinbutton looks odd when flat is applied:

![Bildschirmfoto vom 2022-11-16 11-38-08](https://user-images.githubusercontent.com/19841886/202158846-9241bd48-245b-4d3b-847b-e0ce2dc66bc9.png)

Not sure I like the inconsistency between them. Maybe I misunderstood you, what are your exact suggestions?

**Comment by bertob** [comment]
I agree about keeping the raised styling on the spinbuttons, but everything else could be .flat, I think.

**Comment by flxzt** [comment]
this affects the destructive-action class with the "delete selection", it is ignored when paired with `flat`.

This custom css could be a reasonable fix ( I would only colorize the actual delete icon while keep the grey hover color, but I don't really know how to do that ).
```css
.selection_delete_button {
    color: @destructive_bg_color;
}

.selection_delete_button:hover {
    color: lighter(@destructive_bg_color);
}
```

[Bildschirmaufzeichnung vom 2022-11-16, 13-24-04.webm](https://user-images.githubusercontent.com/19841886/202179930-89a1db4d-71bc-4616-93dc-30f8dfc60413.webm)

 Also I want to make it darker instead of lighter with the light theme, do you know how I can apply css conditionally based on if the theme is currently light / dark?

**Comment by flxzt** [comment]
hm, not sure if I like the flat style with linked toggle buttons. Especially the selector looks / feels a bit weird to me
![Bildschirmfoto vom 2022-11-16 17-49-30](https://user-images.githubusercontent.com/19841886/202290392-9bcd20f3-7f06-4d85-b134-4aa6bf2f9577.png)
